x/evm/migrations/v3/types/params.go

```diff
@@ -0,0 +1,150 @@
+package types
```

Comment:
@crypto-facs How come you need to duplicate the types here? Can't you use `x/evm/types`?